tiptapy/__init__.py

```diff
@@ -58,15 +58,16 @@ class Image(BaseNode):
     def inner_render(self, node) -> str:
         special_attrs_map = {'caption': 'figcaption'}
         attrs = node.get("attrs", {})
-        attrs_s = " ".join(f'{k}="{v}"'
-                           for k, v in attrs.items()
-                           if k not in special_attrs_map
-                           )
+        attrs_s = " ".join(
+            f'{k}="{v}"'
+            for k, v in attrs.items()
+            if k not in special_attrs_map and v
+        )
         html = f"<img {attrs_s}>"
-        for attr in special_attrs_map:
-            if attr in attrs:
-                tag = special_attrs_map[attr]
-                html += f"<{tag}> {attrs[attr]} </{tag}>"
+        # Checking if caption is not empty
+        if attrs['caption'] and 'caption' in attrs:
```

Comment:
@gauravr added a conditional statement and not iterating over `special_attrs_map`

Reply:
@Pradhvan it will raise `key_error` if `caption` is not present in `attrs`